code/modules/mob/living/carbon/human/life.dm

```diff
@@ -1220,6 +1243,32 @@ var/global/list/brutefireloss_overlays = list("1" = image("icon" = 'icons/mob/sc
 						if(ANTAGHUD)
 							process_antag_hud(src)
 
+			if(back && istype(back, /obj/item/weapon/rig)) ///ahhhg so snowflakey

```

Comment:
[Vision/HUD code makes me cry because of how snowflakey it is.](http://i.imgur.com/DC0H2e7.jpg)